M2/Macaulay2/tests/threads/schur-2.m2

```diff
@@ -0,0 +1,5 @@
+needsPackage "SchurRings"
```

Comment:
This new test in this new directory should also be made to run from `tests/Makefile.in`.

Reply:
Done.